Documentation/General/emergentArchitectureDesign/emergentArchitectureDesign.tex

```diff
@@ -95,13 +95,13 @@ \subsection{Moodcat Backend}
 In the following paragraphs we will explain the last two subcomponents.
 
 \subsubsection{REST API}
-The MoodCat REST\cite{rest} connects the frontend to the music matching algorithm.
 The backend keeps track of which rooms are active and what song is currently played in the room.
+It also receives and processes the likes and dislikes, and if disliked the arousal/valence classification.
```

Comment:
'The API can process user votes and classifications in order to improve the system.'